src/swsssdk/port_util.py

```diff
@@ -6,19 +6,22 @@
 
 
 SONIC_ETHERNET_RE_PATTERN = "^Ethernet(\d+)$"
+SONIC_ETHERNET_BP_RE_PATTERN = "^Ethernet-BP(\d+)$"
 SONIC_PORTCHANNEL_RE_PATTERN = "^PortChannel(\d+)$"
 SONIC_MGMT_PORT_RE_PATTERN = "^eth(\d+)$"
 
 
 class BaseIdx:
     ethernet_base_idx = 1
+    ethernet_bp_base_idx = 9000
```

Comment:
>9000 [](start = 27, length = 4)

How do you choose this value? Is 2000 working?

Reply:
What is the range used by a realistic SKU?

---
In reply to: [416935612](https://github.com/Azure/sonic-py-swsssdk/pull/68#discussion_r416935612) [](ancestors = 416935612)

Reply:
Chose 9000 as the offset will be large enough and Index does not overlap. 
With 2000 as offset, it might overlap with PortChannel interface index.

Reply:
@qiluo-msft , i think 9000 should be a good safe offset value for back-plane instead of a lower value